src/otx/core/exporter/base.py

```diff
@@ -64,6 +67,7 @@ def export(
         base_model_name: str = "exported_model",
         export_format: OTXExportFormatType = OTXExportFormatType.OPENVINO,
         precision: OTXPrecisionType = OTXPrecisionType.FP32,
+        dump_auxiliaries: bool = False,
```

Comment:
We may remove this, since custom_forward can be optional, and can be applied only when it's actually passed.

Reply:
I see, it looks like controlling forward is a bit painful in the exporter, so maybe we move it to the OTXModel?
This'd allow overloading exporters with model patching, and also this reduces the depth of modifications, when we need to change something related to model customization.

Reply:
Ok, let me see what I can do

Reply:
Moved it all to OTXModel